api_server.py

```
@@ -205,6 +219,17 @@ def check_rules(available_ids: List[int]) -> Dict:
     }
 
 
+def check_rules(available_ids: List[int]) -> Dict:
+    if not available_ids:
+        return check_rules_from_data([])
+    with get_db() as conn:
+        placeholders = ",".join("?" * len(available_ids))
+        rows = conn.execute(
+            f"SELECT role, skills FROM crew WHERE id IN ({placeholders})", available_ids
```

Ruff (0.15.6)

[error] Possible SQL injection vector through string-based query construction

(S608)